input/templates/partials/catalog/mix-match-product3.json

```diff
@@ -0,0 +1,259 @@
+{
+    "list" : [
```

Comment:
Is it possible to use a `product-list3.json` like we have `product-list4.json` and `product-list9.json`?